lib/npm/install.js

```diff
@@ -18,6 +18,9 @@ function install(context, next) {
   if (context.options.nodedir) {
     var nodedir = path.resolve(process.cwd(), context.options.nodedir);
     options.env.npm_config_nodedir = nodedir;
+    if (process.platform === 'win32') {
+      nodedir = nodedir.replace(/\\/g, '\\\\');
```

Comment:
oh wait... you need to escape both the original and the escaped string... that makes sense.